pkgs/games/anki/bin.nix

```
@@ -0,0 +1,67 @@
+{ fetchurl, stdenv, lib, buildFHSUserEnv, appimageTools, writeShellScript, anki, undmg }:
+
+let
+  pname = "anki-bin";
+  version = "2.1.36";
+
+  tarball = fetchurl {
+    url = "https://github.com/ankitects/anki/releases/download/${version}/anki-${version}-linux.tar.bz2";
+    sha256 = "01xcjnfs5pfh7v0nkffw2wpl19l6pj9k3kxrcawv3cm42asy0mfz";
+  };
+
+  unpacked = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = tarball;
+
+    installPhase = ''
+      runHook preInstall
+
+      xdg-mime () {
+        echo Stubbed!
+      }
+      export -f xdg-mime
+
+      PREFIX=$out bash install.sh
+
+      runHook postInstall
+    '';
+  };
+
+  dmg = fetchurl {
+    url = "https://github.com/ankitects/anki/releases/download/${version}/anki-${version}-mac.dmg";
+    sha256 = "1i6iidm5h8r9g801mvqxi2av03qdw3lr28056fv5ixnb5dq2wqim";
+  };
+
+  meta = with lib; {
```

```suggestion
  meta = with stdenv.lib; {
```
is a bit more common

---

Is there any benefit to that?

---

I'll take @veprbl's approval as permission to mark this as resolved. I hope that's alright :)